recipe/meta.yaml

```
@@ -1,39 +1,45 @@
 {% set name = "pypyodbc" %}
-{% set version = "1.3.5" %}
+{% set version = "1.3.6" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
-  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.2.zip"
-  sha256: 90a9a1fdbe5c06b875155d82562beb50bb22743190ca9b46a2cb71030ee65496
+  # Use GH for LICENSE file
+  url: https://github.com/pypyodbc/pypyodbc/archive/refs/tags/v1.3.6.tar.gz
```

This might be clearer for future updates.
```suggestion
  url: https://github.com/pypyodbc/pypyodbc/archive/refs/tags/v{{ version }}.tar.gz
```

---

Good idea. Thanks !